test/integration/component_guide_test.rb

```diff
@@ -16,23 +17,30 @@ def skip_slimmer
 
 GovukPublishingComponents::ApplicationController.include(GovukPublishingComponentsSkipSlimmer)
 
+# Tests aren't going through Slimmer so we need to explicitly include styles
+GovukPublishingComponents.configure do |c|
+  c.application_print_stylesheet = "core-layout-print"
+  c.application_stylesheet = "core-layout"
```

Comment:
do we need the grid? could we use header-and-footer-only?

Reply:
I'm using the same thing that the guide uses when you view it in a browser.